data/json/items/comestibles/other.json

```diff
@@ -449,6 +449,26 @@
     "vitamins": [ [ "calcium", 4 ], [ "iron", 47 ], [ "vitC", 7 ] ],
     "fun": -10
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "foodplace_food",
+    "name": { "str": "Foodplace's delicious food™", "str_pl": "Foodplace's delicious food™" },
```

Comment:
Shouldn't all these words be capitalized, since it's a branded product?

Reply:
I think I'll keep it like that.